test/client_properties.test.js

```diff
@@ -55,7 +47,7 @@ describe('ClientProperties', () => {
         null,
         [
           ClientPropertiesField.CERT_HOLDER_ID,
-          ClientPropertiesField.CERT_VERSION,
+          ClientPropertiesField.CERT_PEM,
```

Comment:
Is this more like another bug? (or related to the fix?)

Reply:
Yes.
It's related to this fix.
Because `CERT_VERSION` became an optional property that will be assigned with a default value, so I picked another property that will not be assigned with any value to test the "oneOf" capability of the ClientPropertie class.